README.md

```diff
@@ -1,3 +1,7 @@
 # winston
 
 One day, this will be a real README. For now, see [`INSTALL.md`](INSTALL.md).
+
+Possible problems:
+Visit http://openvibe.inria.fr/coadapt-p300-stimulator-tutorial/
```

Comment:
This can (and should) be made into natural language with a link using Markdown. For more information on Markdown, see [the original source](https://daringfireball.net/projects/markdown/).